### DISCUSSION TITLE
小白提问，如何默认加载svpflow和anime4k

### DISCUSSION THREAD

**Discussion by topgaer** [hooke007/mpv_PlayKit#175] [mpv 通用]
懒人包里附带的有，但是每次播放要手动从右键菜单里面选取，关掉重新打开就没有了。
能不能在哪里改成默认加载，或者记住上次从右键界面修改的设置
翻了一下讨论区没有找到，网上搜过，说vcb-s的论坛有修改包，但是论坛已经打不开了

**Comment by hooke007** [comment]
快速说明里有提到 mpv.conf 本身就是设置中心。如果你真的看过就知道这个问题的答案

**Reply by topgaer** [reply]
不好意思，我之前看过了文档，也自己添加了一些东西进去，就是一直不生效，结果发现是我忽视了配置文档最前面的基本说明：#号的意思是代表它后面的东西不生效的，我以为是一种规则，结果自己添加的也写了#，我实在太白了哈哈
